test/check-application

```diff
@@ -32,6 +32,12 @@ DISTROS_WITHOUT_PODMAN_USER_RESTART = ['ubuntu-stable', 'ubuntu-2204']
 DISTROS_WITHOUT_PODMAN_POD_VOLUMES = ['ubuntu-stable', 'ubuntu-2204', 'debian-testing']
 
 
+def podman_version(cls):
+    version = cls.execute(False, "podman -v").strip().split(' ')[-1]
+    # HACK: handle possible rc versions such as 4.4.0-rc2
+    return tuple(int(v.split('-')[0]) for v in version.split('.'))
```

Comment:
If only we had a more machine readable API for determining the version. One should invent an RPC protocol over an Unix socket, and call it "data bus" or something such :thinking: 

(snarky comment aside -- this is ugly, but effective, thanks!)